package.json

```diff
@@ -15,31 +15,43 @@
   },
   "devDependencies": {
     "@semantic-release/exec": "6.0.3",
+    "@tsconfig/node18": "2.0.1",
     "@types/node": "18.16.13",
     "@types/shelljs": "0.8.12",
     "bats": "1.9.0",
     "bats-assert": "2.0.0",
     "bats-support": "0.3.0",
     "commander": "10.0.1",
     "conventional-changelog-conventionalcommits": "5.0.0",
+    "esbuild": "0.17.19",
     "husky": "8.0.3",
     "lint-staged": "13.2.2",
     "npm-run-all": "4.1.5",
+    "pkg": "5.8.1",
     "prettier": "2.8.8",
     "semantic-release": "21.0.2",
-    "shelljs": "0.8.5"
+    "shelljs": "0.8.5",
+    "tsx": "3.12.7",
+    "typescript": "5.0.4"
   },
   "scripts": {
     "test:bats": "node_modules/bats/bin/bats --timing --verbose-run test/bash/ test/bash/v2",
     "test:docker": "node tools/test.js",
     "bats": "node_modules/bats/bin/bats --timing --verbose-run",
-    "build": "release:prepare",
+    "build": "run-s 'build:*'",
+    "build:cli": "node tools/esbuild.js",
+    "start": "tsx src/cli/index.ts",
     "lint": "prettier -c -u \"**/*.*\"",
     "lint-fix": "prettier -w -u \"**/*.*\"",
     "postinstall": "husky install",
     "prepare:proxy": "node tools/prepare-proxy.js",
     "release:prepare": "node tools/prepare-release.js",
     "release:publish": "node tools/publish-release.js"
   },
-  "packageManager": "yarn@3.5.1"
+  "packageManager": "yarn@3.5.1",
+  "dependencies": {
+    "clipanion": "3.2.0",
+    "execa": "7.1.1",
+    "got": "12.6.0"
+  }
```

Comment:
Good to move these above `devDependencies`

Reply:
it doesn't really matter, because we never publish this package. but they are production deps, because thex will be used in the build cli. 🤷‍♂️